### DISCUSSION TITLE
Raspberrymatic erkennt USB Stick nicht mehr?

### DISCUSSION THREAD

**Discussion by ThreeSixty123** [OpenCCU/OpenCCU#2830] [🇩🇪 Fragen&Antworten]
Guten Abend,

nach mehremaligem Neustarten und diversen anderen erfolglosen Versuchen muss ich mich hier mal an euch wenden.

Der Stick wird in Homeassistant zwar angezeigt aber ich kann nichts mehr Steuern bzw. der Status wird nicht gewechselt.

Ich habe das mulmige Gefühl, dass der Stick nicht erkannt wird und im Eimer ist. Kann ich das irgendwie überprüfen?

Hier mal der Log von Raspberrymatic nach dem Neustart.

>***** /var/log/messages *****
Aug 24 23:01:05 de838cd8-raspberrymatic syslog.info syslogd started: BusyBox v1.36.1
Aug 24 23:01:06 de838cd8-raspberrymatic user.info firewall: configuration set
Aug 24 23:01:09 de838cd8-raspberrymatic daemon.err xinetd[1023]: Unable to read included directory: /etc/config/xinetd.d [file=/etc/xinetd.conf] [line=14]
Aug 24 23:01:09 de838cd8-raspberrymatic daemon.crit xinetd[1023]: 1023 {init_services} no services. Exiting...
Aug 24 23:01:10 de838cd8-raspberrymatic user.info root: Updating RF Lan Gateway Coprocessor Firmware
Aug 24 23:01:10 de838cd8-raspberrymatic user.debug update-coprocessor: firmware filename is: coprocessor_update_hm_only.eq3
Aug 24 23:01:10 de838cd8-raspberrymatic user.info root: Updating RF Lan Gateway Firmware
Aug 24 23:01:10 de838cd8-raspberrymatic user.info update-lgw-firmware: No gateway found in config file /etc/config/rfd.conf
Aug 24 23:01:45 de838cd8-raspberrymatic daemon.info : starting pid 1368, tty '': '/bin/mv /tmp/boot.log /var/log/boot.log'
Aug 24 23:01:45 de838cd8-raspberrymatic daemon.info : starting pid 1369, tty '/dev/null': '/usr/bin/monit -Ic /etc/monitrc'
Aug 24 23:01:45 de838cd8-raspberrymatic user.info monit[1369]: Starting Monit 5.33.0 daemon with http interface at /var/run/monit.sock
Aug 24 23:01:45 de838cd8-raspberrymatic user.info monit[1369]: 'de838cd8-raspberrymatic' Monit 5.33.0 started
Aug 24 23:02:00 de838cd8-raspberrymatic user.err monit[1369]: 'udevdEnabled' status failed (0) -- no output
Aug 24 23:02:00 de838cd8-raspberrymatic local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000420C9A186E5:0","CARRIER_SENSE_LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: CARRIER_SENSE_LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Aug 24 23:02:00 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"000420C9A186E5:0","CARRIER_SENSE_LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1445]
Aug 24 23:02:00 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: 'sshdEnabled' status failed (1) -- no output
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: 'hs485dEnabled' status failed (2) -- grep: /var/etc/hs485d.conf: No such file or directory
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: 'hmlangwEnabled' status failed (1) -- no output
Aug 24 23:02:01 de838cd8-raspberrymatic user.warn monit[1369]: 'hasUSB' status failed (1) -- no output
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: 'hb_rf_eth-CheckEnabled' status failed (1) -- no output
Aug 24 23:02:01 de838cd8-raspberrymatic user.warn monit[1369]: 'rpi4usb3Check' status failed (1) -- no output
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: 'tailscaleEnabled' status failed (1) -- no output
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: Lookup for '/media/usb1' filesystem failed  -- not found in /proc/self/mounts
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: Filesystem '/media/usb1' not mounted
Aug 24 23:02:01 de838cd8-raspberrymatic user.err monit[1369]: 'usb1' unable to read filesystem '/media/usb1' state
Aug 24 23:02:01 de838cd8-raspberrymatic user.info monit[1369]: 'usb1' trying to restart
Aug 24 23:02:17 de838cd8-raspberrymatic user.warn monit[1369]: 'hasUSB' status failed (1) -- no output
Aug 24 23:02:17 de838cd8-raspberrymatic user.warn monit[1369]: 'rpi4usb3Check' status failed (1) -- no output
Aug 24 23:02:17 de838cd8-raspberrymatic user.err monit[1369]: Filesystem '/media/usb1' not mounted
Aug 24 23:02:17 de838cd8-raspberrymatic user.err monit[1369]: 'usb1' unable to read filesystem '/media/usb1' state
Aug 24 23:02:17 de838cd8-raspberrymatic user.info monit[1369]: 'usb1' trying to restart
Aug 24 23:02:33 de838cd8-raspberrymatic user.warn monit[1369]: 'hasUSB' status failed (1) -- no output
Aug 24 23:02:33 de838cd8-raspberrymatic user.err monit[1369]: 'rpi4usb3Check' status failed (1) -- no output
Aug 24 23:02:33 de838cd8-raspberrymatic user.err monit[1369]: Filesystem '/media/usb1' not mounted
Aug 24 23:02:33 de838cd8-raspberrymatic user.err monit[1369]: 'usb1' unable to read filesystem '/media/usb1' state
Aug 24 23:02:33 de838cd8-raspberrymatic user.info monit[1369]: 'usb1' trying to restart
Aug 24 23:02:49 de838cd8-raspberrymatic user.warn monit[1369]: 'hasUSB' status failed (1) -- no output
Aug 24 23:02:49 de838cd8-raspberrymatic user.err monit[1369]: Filesystem '/media/usb1' not mounted
Aug 24 23:02:49 de838cd8-raspberrymatic user.err monit[1369]: 'usb1' unable to read filesystem '/media/usb1' state
Aug 24 23:02:49 de838cd8-raspberrymatic user.info monit[1369]: 'usb1' trying to restart
Aug 24 23:03:04 de838cd8-raspberrymatic user.err monit[1369]: 'hasUSB' status failed (1) -- no output
***** /var/log/lighttpd-error.log *****
2024-08-24 23:01:09: (../src/server.c.1939) server started (lighttpd/1.4.76)
2024-08-24 23:01:44: (../src/server.c.1230) [note] graceful shutdown started
2024-08-24 23:01:44: (../src/server.c.1181) [note] pid 1367 continuing to handle 0 connection(s) in progress
2024-08-24 23:01:45: (../src/server.c.1939) server started (lighttpd/1.4.76)
2024-08-24 23:01:44: (../src/server.c.1230) [note] graceful shutdown started
2024-08-24 23:01:44: (../src/server.c.2350) server stopped by UID = 0 PID = 1037

Könnt ihr mir da Helfen??

**Comment by Baxxy13** [comment]
> in Homeassistant

Das ist viel zu pauschal. 
Welche Hardware? (Raspi/PC/VM etc.)
Homeassistant-OS/Supervised/Docker/etc.?
RaspberryMatic als Homeassistant-AddOn?

Mal davon ausgehend das Homeassistant-OS mit dem RaspberryMatic-AddOn genutzt wird...
Welche OS-Version?
Bei HA-OS 13.x liest man viele Probleme die sich auf USB-Sticks beziehen, könnte auch hier der Fall sein.

**Reply by ThreeSixty123** [reply]
Hi sorry war schon was spät. 

Hardware ist: Raspberry Pi 4 mit Homeassistant OS und Supervisor. Add On ist Raspberry Matic. 

Homeassistant OS ist 
Core 2024.8.2
Supervisor 2024.08.0
Operating System 13.1
Frontend 20240809.0

Komischerweise läuft nun alles wieder … :( ?!

Dies ist schon das 3. Mal, dass der Funkstick sich durch irgendwelche ( vorher auch dagewesenen anderen Frequenzen ) auf einmal gestört fühlt. 


Ist zwar alles eine nette Sache mit Homeassistant und dem Homematic Ip usb Stick aber irgendwie doch recht anfällig
